src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseChecklistService.cs

```diff
@@ -159,7 +160,7 @@ private async Task<bool> ReleaseSectionHasEmptyHtmlBlock(Guid releaseId, Content
                     cb.ContentSection!.Release.ReleaseId == releaseId &&
                     cb.ContentSection.Type == sectionType)
                 .OfType<HtmlBlock>()
-                .AnyAsync(htmlBlock => string.IsNullOrEmpty(htmlBlock.Body));
+                .AnyAsync(htmlBlock => htmlBlock.Body.IsNullOrEmpty());
```

Comment:
`StringExtensions.IsNullOrEmpty` can't be translated to SQL. I've not tested it but I believe this change will introduce a new bug that no analyst will be able to approve a release as this will throw an exception when calculating the checklist.

Reply:
Tested this now by running this branch locally, and I'm afraid it's bad news 😢 

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/4147126/51783951-4381-4552-a7ea-7fe394f64670)

Reply:
I'll just restore them all.

Reply:
Done

Reply:
Ok cool, will approve. Fyi, I was only aware of this being a problem in `ReleaseChecklistService`. It was probably fine in the other cases. The difference is whether or not the expression needs to be translated by LINQ to SQL.